commands/test/script

```diff
@@ -75,5 +30,16 @@ setup-test-hacks
     full)
       make test-full
       ;;
+    scaleup)
```

Comment:
Also note this is new - for all you people wanting to do the scale up/down test in the release spreadsheet:

`./rigger test --type scaleup`

will do the trick